src/clusterfuzz/_internal/fuzzing/corpus_manager.py

```diff
@@ -259,12 +280,39 @@ def rsync_from_disk(self,
     """
     corpus_gcs_url = self.get_gcs_url()
     legalize_corpus_files(directory)
-    result = self._gsutil_runner.rsync(directory, corpus_gcs_url, timeout,
-                                       delete)
+    result = self._gsutil_runner.rsync(
+        directory, corpus_gcs_url, timeout, delete=delete)
+
+    # Upload zipcorpus.
+    # TODO(metzman): Get rid of the rest of this function when migration is
+    # complete.
+    filenames = shell.get_files_list(directory)
+    self._upload_to_zipcorpus(filenames, partial=False)
 
     # Allow a small number of files to fail to be synced.
     return _handle_rsync_result(result, max_errors=MAX_SYNC_ERRORS)
 
+  def get_zipcorpora_gcs_urls(self, max_partial_corpora=float('inf')):
```

Comment:
nit: docstring.

Reply:
Also add more comments about what this is doing: This is getting the N (latest?/random?) partial corpora.